templates/politician/politician_edit.html

```diff
@@ -141,8 +141,8 @@ <h1>
         
         <input type="text" name="profile_image_background_color" id="profile_image_background_color_id" class="form-control"
                value="{% if politician %}{{ politician.profile_image_background_color|default_if_none:"" }}{% else %}{{ profile_image_background_color|default_if_none:"" }}{% endif %}" />
-        <label>Regenerate default color &nbsp;</label><input name="regenerate_color" type="checkbox"/>
-        <label>Regenerate default color &#40;different algorithm&#41;  &nbsp;</label><input name="regenerate_color_edge_case" type="checkbox"/>
+        <input name="regenerate_color" type="checkbox"/><label> &nbsp; Regenerate default color</label>
```

Comment:
Hi @brandta-1 could you please give the input an id (<INPUT_ID>), and then add `for="<INPUT_ID>"` to the label? That way, clicking on the label will control the checkbox.